doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2075,11 +2075,13 @@ in :ref:`canonicalstructures`; here only a simple example is given.
       This is equivalent to a regular definition of :token:`ident` followed by the
       declaration :n:`Canonical @ident`.
 
-.. cmd:: Print Canonical Projections
+.. cmd:: Print Canonical Projections {* @ident}
 
    This displays the list of global names that are components of some
    canonical structure. For each of them, the canonical structure of
-   which it is a projection is indicated.
+   which it is a projection is indicated. If constants are given as
+   its arguments, only the unification rules that involve or are
+   synthesized from given constants will be shown.
```

Comment:
I suggest the following
```suggestion
   synthesized from simultaneously all given constants will be shown.
```
to disambiguate that all constraints must be satisfied and not at least one.